app/javascript/mastodon/components/load_gap.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useCallback } from 'react';
```

Comment:
```suggestion
import { useCallback } from 'react';
```

The `React` import is no longer needed since 25093

Reply:
Fixed. Thank you.